tango_simlib/sim_test_interface.py

```diff
@@ -3,28 +3,35 @@
 #                                                                                       #
 # BSD license - see LICENSE.txt for details                                             #
 #########################################################################################
+from __future__ import absolute_import, division, print_function
+
 import weakref
 
+from future import standard_library
+from future.utils import with_metaclass
 from tango import Attr, AttrWriteType, DevDouble, DevState, UserDefaultAttrProp
 from tango.server import Device, DeviceMeta, attribute, device_property
 from tango_simlib import model
 from tango_simlib.utilities.helper_module import generate_cmd_handler
 
+standard_library.install_aliases()
+
+
 
-class TangoTestDeviceServerBase(Device):
-    __metaclass__ = DeviceMeta
```

Comment:
@tockards and @mmphego why was this removed?

Reply:
It was not removed by an individual but by futurize stage 2.

See: https://python-future.org/compatible_idioms.html?highlight=abc%20metaclass#metaclasses

If indoubt what metaclasses are, check this video out: https://www.youtube.com/watch?v=0NFFk32CJ8s